apps/OpenSign/public/locales/fr/translation.json

```diff
@@ -0,0 +1,591 @@
+ {
+    "create-account": "Créer un compte",
+    "login": "se connecter",
+    "language":"Langue",
+    "name":"Nom",
+    "phone":"Téléphone",
+    "email":"E-mail",
+    "company":"Entreprise",
+    "job-title":"Titre d'emploi",
+    "profile":"Profil",
+    "is-email-verified":"L'e-mail est-il vérifié",
+    "log-out" :"Déconnexion",
+    "public-profile":"Profil public",
+    "tagline":"Slogan",
+    "has-requested-you-to-sign":" vous a demandé de signer",
+    "has-requested-you-to-review-and-sign":"vous a demandé de consulter et de signer",
+     "input-required":"Veuillez remplir ce champ.",
+    "sort-order" :{
+      "Ascending":"Ascendant",
+      "Descending":"Descendant",
+      "Name":"Nom",
```

Comment:
"name":"Nom et Prénom",

User can missunderstand and only write his Name and not firstname